deepmd/pt/model/model/__init__.py

```diff
@@ -140,7 +140,8 @@ def get_standard_model(model_params):
     fitting_net["type"] = fitting_net.get("type", "ener")
     fitting_net["ntypes"] = descriptor.get_ntypes()
     fitting_net["mixed_types"] = descriptor.mixed_types()
-    fitting_net["embedding_width"] = descriptor.get_dim_emb()
+    if fitting_net["type"] != "dos":
+        fitting_net["embedding_width"] = descriptor.get_dim_emb()
```

Comment:
it seems that `embedding_width` is only needed by polar and dipole fitting. 
```python 
if fitting_net["type"]  in ["dipole", "polar"]:
    fitting_net["embedding_width"] = descriptor.get_dim_emb()
```
may be bettter